packages/grafana-schema/src/schema/dashboard/v2alpha0/dashboard.gen.ts

```diff
@@ -801,6 +801,10 @@ export type VariableType = "query" | "adhoc" | "groupby" | "constant" | "datasou
 
 export const defaultVariableType = (): VariableType => ("query");
 
+export type VariableKind = QueryVariableKind | TextVariableKind | ConstantVariableKind | DatasourceVariableKind | IntervalVariableKind | CustomVariableKind | GroupByVariableKind | AdhocVariableKind;
```

Comment:
I found this type very useful and semantically, it is great to have a kind that includes all variables.